pkg/controller/podautoscaler/horizontal.go

```diff
@@ -146,10 +143,65 @@ func (a *HorizontalController) Run(stopCh <-chan struct{}) {
 		return
 	}
 
+	// start a single worker (we may wish to start more in the future
+	go wait.Until(a.worker, time.Second, stopCh)
```

Comment:
Once from line 141 (WaitForCacheSync()) and once from line 147 (wait.Until()). These are my feelings and I may be wrong.